### PR TITLE
Add documentation for DependencyAdder DSL extensions

### PR DESCRIPTION
**Pull request by big-guy**

### CODE REVIEW

**Review by octylFractal — APPROVED** [review]

**Comment by big-guy** [comment]
@bot-gradle test and merge

File: subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/DependencyAdderExtensions.kt [hunk 1]
```diff
@@ -32,6 +32,32 @@ import org.gradle.api.file.FileCollection
...
+ *
+ * In this Kotlin DSL example, {@code implementation("org:foo:1.0")}
+ * <ul>
+ *     <li>is equivalent to {@code getImplementation().invoke("org:foo:1.0")} because of these extension functions</li>
+ *     <li>has the same effect as {@code getImplementation().add("org:foo:1.0")} in Java</li>
+ * </ul>
+ *
+ * There are {@code invoke(...)} equivalents for all the {@code add(...)} methods in {@code DependencyAdder}.
+ *
+ */
```
Comment (blindpirate) [inline subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/DependencyAdderExtensions.kt:60]:
```suggestion
 */
 
```

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

File: subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/DependencyAdderExtensions.kt (hunk 1 above)
Comment (big-guy) [reply subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/DependencyAdderExtensions.kt:60]: Thanks Bo

**Pull request merged**